open-company/open-company-email
# Email digest refresh

Card: https://trello.com/c/m1zu3mUW

To test:
- add a few posts (make sure you have an MS, posts on multiple boards and there are coments and interactions)
- send yourself an email digest
- [x] digest checks in with the design from the card? Good
- [x] make sure seen/unseen are also respected
- [x] make sure MS are always up top and the rest is grouped by board and sorted by publish date
